src/components/Comments/ReplyCommentItem.js

```diff
@@ -44,7 +46,8 @@ const ReplyCommentItem = ({
   disableOnTextPress = false,
   refreshComment,
   updateVoteParent,
-  onLongPress
+  onLongPress,
+  feedId
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `feedId` prop has been added to the component. Ensure that this prop is being passed correctly from the parent components and it's not undefined or null where it's used.



<!-- This is an auto-generated comment by OSS CodeRabbit -->